test/test_multiprocessing.py

```diff
@@ -269,7 +269,11 @@ def test_receive():
             t1 = q.get()
             t2 = q.get()
             self.assertTrue(t1.eq(1).all())
-            self.assertTrue(id(t1.storage()) == id(t2.storage()))
```

Comment:
At the moment, I'm not sure how to make subsequent `Tensor.storage()` calls return storages with identical IDs. Is this a BC break that anyone would care about?

Reply:
It's most likely to matter if someone is doing a hash map keyed on storages. I agree this is difficult to do, let us break it and pray.